src/services/effects.ts

```diff
@@ -19,13 +19,19 @@ export function withEffects(store: Store<Actions>) {
     .subscribe(async ([distribution, serviceAreas]) => {
       let points = await getRepresentativePoints(serviceAreas)
 
+      // JavaScript doesn't preserve zero points for numbers. Because
+      // `population` keys are strings in ("0.5", "2.5", "5.0"), when
+      // `distribution` is 5 we need to manually convert it to the string
+      // "5.0" so that we can check it against its corresponding key.
+      let _distribution = distribution.toFixed(1)
```

Comment:
Nice catch

Reply:
Thanks - not sure why it worked for me before.